datafusion/substrait/tests/cases/substrait_validations.rs

```diff
@@ -104,20 +103,18 @@ mod tests {
             );
             // the DataFusion schema { b, a, c, d } contains the Substrait schema { a, b, c }
```

Comment:
nit, given you changed the order (to make the test more powerful, I think?):
```suggestion
            // the DataFusion schema { d, a, c, b } contains the Substrait schema { a, b, c }
```

Reply:
missed this. yes, before this, the naive wrong implementation yielded [b, a] which looked way too similar to [a, b]. now it will be [d, a].